### ISSUE TITLE
feat: fresh object per-view

### ISSUE THREAD

**Issue by mosheavni** [Ramilito/kubectl.nvim#464]
right now the option to color fresh objects is hard-coded only on some views, and is global for all views.
for example, today I really needed to see fresh nodes (since i'm rolling many nodes)

- add "fresh object" wherever makes sense, maybe add option to include on a per-view basis
- add the option to change the "fresh" time on a per-view basis

**Comment by Ramilito** [comment]
> add the option to change the "fresh" time on a per-view basis

I don't think we should do that, don't feel that it's going to be used that way.

> add "fresh object" wherever makes sense, maybe add option to include on a per-view basis

My idea with the fresh object is to be an opt-in per view, similar to what you did with the nodes view
